product_owners.md

```diff
@@ -0,0 +1,27 @@
+# Product Owners
+
+We strive for each of our applications to have a product owner (PO). The product
+owner is usually a key stakeholder for the application who can collaborate
+closely with us to guide development. Documentation of our team's [work cycles](/work_cycles.md) and [technical liaison role](/technical_liaisons.md) should give product owners useful context for this collaboration.
+
+The exact practices of each product owner will vary according to the application and the
+individuals themselves. Generally, product owners will work to:
+
+* Communicate with other stakeholders and users as-needed, either ad-hoc as a committee.
+* Participate during relevant work cycles by attending the planning meeting and
+  Friday wrap-up meeting when the PO's application is a focus of the cycle.
+* Work with the technical liaison to prioritize and specify issues. Having a GitHub account will enable the PO to be most fully integrated into this process.
+* Be available on slack during a relevant work cycle to answer questions and
+  give feedback.
+
+DLS will work to:
+
+* Provide appropriate notice to a PO when a relevant work cycle is upcoming, at least 1 week before it begins, and earlier if possible.
```

Comment:
Okay, so this means 1 week before the beginning of the maintenance & research week, i.e. 1 week before the planning meeting.

Reply:
I think it might be helpful to spell it out a bit more, even though the "work cycles" document is linked.